slt/standard/functions/scalars/string/left.slt

```diff
@@ -0,0 +1,43 @@
+# LEFT
+
+query T
+SELECT left('hello', 3);
+----
+hel
+
+query T
+SELECT left('hello', 5);
+----
+hello
+
+query T
+SELECT left('hello', 10);
+----
+hello
+
+query T
+SELECT left('hello', 0);
+----
+(empty)
+
+query T
+SELECT left('hello', -5);
+----
+(empty)
```

Comment:
Add an additional test case `SELECT left('hello', -3)` which should return 'he'.